### PR TITLE
Minor fix, updated maintainers, and changed homepage location to redhat-developer

### PR DESCRIPTION
**Pull request by ianrhamilton**

### CODE REVIEW

**Comment by ianrhamilton** [comment]
@LightGuard could you please merge? I don't have the correct privileges. Added myself as a maintainer, and changed homepage location to redhat-developer/blinkr. Also includes a small fix to a bug in blinkr as it was chomping index.html of links that needed it causing false failures. This has been fixed for a month or so (we're still pointing broken link checks to my fork).

I'm beginning to start preparing this code for a release to rubygems. I just need to make a few tweaks, and also update the README file.

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,9 @@ test/tmp
 test/version_tmp
 tmp
 *.swp
+.idea/.rakeTasks
```
Comment (LightGuard) [inline .gitignore]: I think simply ignoring `.idea` should be enough

File: blinkr.gemspec [hunk 2]
```diff
@@ -7,14 +7,14 @@ require 'blinkr/version'
 Gem::Specification.new do |spec|
   spec.name = 'blinkr'
   spec.version = Blinkr::VERSION
-  spec.authors = ['Pete Muir', 'Jason Porter']
-  spec.email = %w[pmuir@bleepbleep.org.uk lightguard.jp@gmail.com]
+  spec.authors = ['Ian Hamilton', 'Jason Porter']
```
Comment (LightGuard) [inline blinkr.gemspec]: Pete was still the original author, he should still be listed.

**Review by LightGuard — CHANGES_REQUESTED** [review]

**Comment by ianrhamilton** [comment]
@LightGuard thanks for the review. Pushed changed to address your comments.

**Comment by ianrhamilton** [comment]
@LightGuard is there anymore changes I need to make to this?

**Pull request merged**